var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -62,21 +63,23 @@ def libs(self):
 
     build_directory = 'spack-build'
 
-    def setup_environment(self, spack_env, run_env):
-        # TODO: set optimum flags for platform+compiler combo, see
```

Comment:
why remove comment?

Reply:
Oh, I just removed the entire function. I added some optimization flags now and kept the comments.